doto-frontend/src/components/ModalContent.js

```diff
@@ -38,6 +38,7 @@ const ModalContent = props => {
     const [selectedName, setSelectedName] = useState("TASK - " + new Date());
     const [selectedDescription, setSelectedDescription] = useState("");
     const [selectedDueDate, setSelectedDueDate] = useState(new Date());
+    const [selectedEarliestDate, setEarliestDate] = useState(new Date());
```

Comment:
I was wondering if you could combine this into a single line, as they are both using `useState(new Date());` or will JavaScript complain?

Reply:
Tbh I'm not too familiar with React states but I believe they have to separate consts? (Just based on looking at the rest of the code with other states)

Reply:
No worries about it then.